warehouse/models/staging/gtfs_guidelines/stg_gtfs_guidelines__no_rt_critical_validation_errors.sql

```diff
@@ -0,0 +1,67 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('stg_gtfs_guidelines__feed_guideline_index') }}
+    WHERE check = {{ no_rt_critical_validation_errors() }}
+),
+
+-- gtfs_rt_fact_files_wide_hourly has one row per day per ID+URL
```

Comment:
I see this comment repeated -- I think it may be worth updating to say `one row per day per ID+URL+feed type` (or clarifying what is meant by URL, normally I would just interpret as `calitp_url_number`, in which case this is not quite accurate)

Reply:
this makes more sense, thanks!